test/dynamo/test_python_autograd.py

```diff
@@ -204,7 +204,7 @@ def _common(self, fn, expected_ops):
         args1 = [torch.randn(10), torch.randn(10)]
         args2 = [torch.randn(10), torch.randn(10)]
         cnt = CompileCounter()
-        fn_dynamo = torch._dynamo.optimize_assert(cnt)(fn)
+        fn_dynamo = torch._dynamo.optimize(cnt)(fn)
```

Comment:
This allows graph break (optimize_assert is the same as nopython=True).  Is that intentional?